bin/prepare-mochitests-dev

```diff
@@ -17,17 +38,31 @@ if [ $? -ne 0 ]; then
   exit -1
 fi
 
-# Clean, package, and copy the activity stream files, default SYMLINK_TESTS to
-# true but allowing the environment to override so that calling scripts like
-# test-merges.js can export real files in order to have hg add them correctly
-# during the export phase.
+# blow away any old bits in order to workaround bug 1335976 for users
+# who are using the default objdir-frontend
+rm -f ${FIREFOX_PATH}/objdir-frontend/dist/bin/browser/features/@activity-streams/*
```

Comment:
This shouldn't really be needed going forwards as `moz.build` for the `system-addon` export does not include any directories as part of `FINAL_TARGET_FILES` but this does help for now.